packages/client/src/AccountsClient.js

```diff
@@ -269,6 +301,18 @@ const Accounts = {
   refreshSession(): Promise<void> {
     return this.instance.refreshSession();
   },
+  verifyEmail(token: string): Promise<void> {
+    return this.instance.verifyEmail(token);
+  },
+  resetPassword(token: string, newPassword: string): Promise<void> {
+    return this.instance.resetPassword(token, newPassword);
+  },
+  requestPasswordReset(userId: string, email?: string): Promise<void> {
```

Comment:
Implementation question: Is it necessary to have `userId` passed in here? This code is being called on the client who isn't signed in, where does the userId come from?